gui-tk/wiring.tcl

```diff
@@ -524,6 +524,9 @@ proc doubleButton {x y} {
         selectVar $x $y
         editItem
     }
+    
+    # For ticket 1092. Reinstate delete handle user interaction
```

Comment:
Can we be sure that you won't select an item and a handle at the same time? It might be safer to have delHandle con ditioned in the else part of this conditional.

Reply:
Thanks, I've put it in the else part.